drizzlepac/devutils/comparison_tools/compare_sourcelist_flagging.py

```diff
@@ -130,27 +145,62 @@ def make_regions(sl_name, shape, color):
     n_sources : int
         total number of sources in sourcelist
     """
-    table_data = Table.read(sl_name, format='ascii.ecsv')
+    if sl_name.endswith(".ecsv"):
+        table_data = Table.read(sl_name, format='ascii.ecsv')
+    elif (sl_name.endswith("daophot.txt") or sl_name.endswith("daophot_corrected.txt")):
+        table_data = Table.read(sl_name, format='ascii')
+    elif (sl_name.endswith("sexphot.txt") or sl_name.endswith("sexphot_corrected.txt")):
+        table_data = Table.read(sl_name, format='ascii')
+    else:
+        sys.exit("ERROR! Unrecognized catalog filetype!")
     flag_counts = np.zeros(9, dtype=int)
     reg_dict = {}
     for bit_val in bit_list:
         reg_dict[bit_val] = ""
-
+        flag_col_name = "Flags"
     for table_line in table_data:
         if sl_name.endswith("point-cat.ecsv"):
-            x = table_line["X-Center"]
-            y = table_line["Y-Center"]
+            if coordsys == "xy":
+                x = table_line["X-Center"]
+                y = table_line["Y-Center"]
+            if coordsys == "radec":
+                x = table_line["RA"]
+                y = table_line["DEC"]
```

Comment:
It may be simpler to define a dictionary for this definitions; like:
```
    table_cols = {'point-cat.ecsv':{'xy':('X-Center', 'Y-Center'), 'radec':('RA','DEC')}
    # table_cols would need entries for all the cases you want to check for.  
    # These definitions would then be accessed in the following way...
    for name in table_cols:
        if sl_name.endswith(name):  
            x,y = table_cols[name][coordsys]
            break
```
This would simplify the next dozen or so lines and make it a little easier to maintain (I think).  Only a suggestion, and not something which needs to be changed.

Reply:
issue resolved by commit [28b39da](https://github.com/spacetelescope/drizzlepac/pull/611/commits/28b39da6532bead3322c480cf065fdfb9e206fde)